celest/coordinates/astronomical_quantities.py

```diff
@@ -0,0 +1,602 @@
+
+
+from celest.units.quantity import Quantity
+from celest import units as u
+from celest.constants import (
+    JD2000_DATE,
+    DAYS_IN_JULIAN_CENTURY,
+    EARTH_ROTATION_RATE_DEG_PER_DAY,
+    ERA_AT_JD2000_DEG
+)
+from typing import Tuple
+import numpy as np
+
+
+def earth_rotation_angle(julian: Quantity) -> Quantity:
+    """Return Earth rotation angle in decimal degrees.
```

Comment:
It doesn't have to be degrees. Update the single line documentation everywhere to reflect this.